src/redux/reducer.js

```diff
@@ -174,6 +175,14 @@ function moreMenu(state = initialState.moreMenu, action) {
       return Object.assign({}, state, {expanded: true});
     case actType.COLLAPSE_MORE_MENU:
       return Object.assign({}, state, {expanded: false});
+    case actType.INCREMENT_FONT_SIZE:
+      return (state.fontSize === 56 ?
+              Object.assign({}, state) :
+              Object.assign({}, state, {fontSize: state.fontSize + 4}));
+    case actType.DECREMENT_FONT_SIZE:
+      return (state.fontSize === 8 ?
+              Object.assign({}, state) :
+              Object.assign({}, state, {fontSize: state.fontSize - 4}));
```

Comment:
The === checks are fragile since they rely on incrementation by multiple of 4. I would use >= and <= instead to enforce bounds. Or even better would be to use Math.max and Math.min. Also you should put the boundary numbers in constants.js